packages/components/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 5.0.0 (Unreleased)
```

Comment:
Not sure I'd count this as a `5.0.0` feature; it's more like `4.1.0`. It isn't a breaking change and it's just an addition.

Reply:
Fair point :)